src/components/templates/PosterPage/components/PosterPageSettingsControl/PosterPageSettingsControl.tsx

```diff
@@ -12,59 +12,44 @@ import "./PosterPageSettingsControl.scss";
 
 export interface PosterPageSettingsControlProps {
   venueId: string;
-  isPosterLive?: boolean;
+  isPosterLive: boolean;
 }
 
 export const PosterPageSettingsControl: React.FC<PosterPageSettingsControlProps> = ({
-  isPosterLive = false,
```

Comment:
This change of optional prop + removing the default should be reverted. I've described this in more detail on the previous comment thread about it, but basically since you removed the 'inner component' of the modal, it made my suggested comments irrelevant, and the code was correct before the changes made that implemented this.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1299/commits/ce562c6a4616118ea904bb8f088ca751787a5f95